pages/resources/index.vue

```diff
@@ -128,7 +119,9 @@ export default {
      * @returns {Number}
      */
     currentResourceCount: function() {
-      return this.resourceData.skip !== 0
+      return this.resourceData.limit > this.resourceData.total
```

Comment:
This looks like a bad merge since it's returning two statements.

Reply:
No the logic here is correct. The platform tab for example only has 11 resources instead of 12. This checks if the limit of the data returned is larger than that of the total result set, and if it is than it should return the total number instead of the limit. So instead of it saying `Showing 12 of 11 resources` it says `showing `11 of 11 resources`. You can chain multiple `else if` blocks in tierary operators which is why it looks weird here.

Reply:
That's interesting, I didn't know that.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator 

Yeah just learned it the other day! Pretty cool 👍